src/components/map/Mapbox.vue

```diff
@@ -49,11 +66,41 @@ export default {
         center: [-123.120735, 49.28273],
         zoom: 12
       },
+      searchQuery: "",
+      suggestions: []
     };
   },
 
   created() {
     this.map = null;
+    this.geocoderControl = null;
+  },
+  mounted() {
+    // EventBus.$on("results", event => {
+    //   console.log("payload: ", event);
+    //   this.defaultInput = event;
+    // });
+    // this.geocoderControl = new MapboxGeocoder({
+    //   accessToken: this.accessToken
+    // });
+    // console.log(this.geocoderControl);
+    // this.geocoderControl.addTo("#map");
+    // this.geocoderControl.on("loading", data => {
+    //   console.log(data);
+    // });
+    // this.geocoderControl.on("results", results => {
+    //   console.log(results.features);
+    //   EventBus.$emit("sendSuggestions", results.features);
+    //   // const input = results.query ? results.query.join("") : "";
+    //   // this.$emit("update:input", input);
+    //   // this.map.flyTo({
+    //   //   center: [
+    //   //     -74.5 + (Math.random() - 0.5) * 10,
+    //   //     40 + (Math.random() - 0.5) * 10
+    //   //   ],
+    //   //   essential: true // this animation is considered essential with respect to prefers-reduced-motion
+    //   // });
+    // });
```

Comment:
Delete if not used?

Reply:
I kinda wanna keep this code but I'll keep it somewhere else. This is to get started with the other geocoding method (building it from scratch, which we might need later on).